pytket/setup.py

```diff
@@ -190,21 +190,22 @@ def finalize_options(self):
     license="Apache 2",
     packages=setuptools.find_packages() + ["pytket.qasm.includes"],
     install_requires=[
-        "sympy ~=1.6",
-        "numpy >=1.21.4, <2.0",
-        "lark ~=1.1",
-        "scipy ~=1.13",
+        "sympy >= 1.12.1",
+        "numpy >= 1.26.4",
```

Comment:
Have you tested if everything still works with 1.26? What is our future testing plan? When using feature only available in 2.0 the ci would not show any problems?

Reply:
Yes I have checked this locally.

This is a question we face already with other packages (e.g. how do we know it still works with sympy 1.6). I don't think there's an easy answer; we can't test every version of everything. This is part of the trade-off between full determinism and compatibility... But from what I've seen of the changes in numpy 2.0, I think it's unlikely.